lib/IDE/ExprContextAnalysis.cpp

```diff
@@ -91,15 +91,35 @@ void typeCheckContextImpl(DeclContext *DC, SourceLoc Loc) {
 } // anonymous namespace
 
 void swift::ide::typeCheckContextUntil(DeclContext *DC, SourceLoc Loc) {
-  // The only time we have to explicitly check a TopLevelCodeDecl
-  // is when we're directly inside of one. In this case,
-  // performTypeChecking() did not type check it for us.
+  // Lookup the swift module.  This ensures that we record all known
+  // protocols in the AST.
+  (void) DC->getASTContext().getStdlibModule();
+
+  bindExtensions(*DC->getParentSourceFile());
```

Comment:
Maybe the getStdlibModule() and bindExtensions() calls should be done as part of performParseAndNameBindingOnly()?

Reply:
Made `bindExtensions()` a part of `performParseAndNameBindingOnly()`.

It turned out that `getStdlibModule()` is meaningless. Since `ASTContext::getStdlibModule(bool loadIfAbsent = false)` is called with default `loadIfAbsent = false`, it doesn't do anything. So I just removed it from `performTypechecking()`